DFC.ServiceTaxonomy.Taxonomies/Controllers/AdminController.cs

```diff
@@ -47,7 +48,8 @@ public class AdminController : Controller
             IEnumerable<ITaxonomyTermValidator> validators,
             IEnumerable<ITaxonomyTermDeleteValidator> deleteValidators,
             IEnumerable<ITaxonomyTermHandler> handlers,
-            ITaxonomyHelper taxonomyHelper)
+            ITaxonomyHelper taxonomyHelper,
+            ITaxonomyTermPublishHandler taxonomyTermPublishHandler)
```

Comment:
Could we add a new method on `ITaxonomyTermHandler` instead?
And could we call it `Published` to indicate it's handling an event, rather than `Publish`, which sounds more like an instruction.
It should also accept an `IEnumerable<IContentOrchestrationHandler>`, in case we want to add any more handlers later.